deploy-prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+cd /var/www/dds
+
+# Update mes-aides
+git fetch origin
+git rebase origin/master
+npm install
+grunt build
+
+# Update openfisca
+npm run install-openfisca
+cat openfisca/api_config.ini | sed s/"port = 2000"/"port = 12000"/ > ~/production.ini
+
+# Restart mes-aides
+sudo restart dds
+
+# Restart openfisca
+sudo start openfisca || sudo restart openfisca
```

Comment:
What I meant was fixed in https://github.com/sgmap/mes-aides-ui/pull/390/commits/fcfad8e3a322d317c2d08a9c67ec500fdbe337b5 ;)